src/common/os_deep_persist_linux.c

```diff
@@ -78,6 +78,37 @@ os_deep_flush_write(int region_id)
 	return 0;
 }
 
+/*
+ * os_deep_persist_type -- perform deep persist operation based on a pmem
+ * mapping type
+ */
+static int
+os_deep_persist_type(const struct map_tracker *mt, void *addr, size_t len)
+{
+	ASSERTne(mt->type, MAX_PMEM_TYPE);
+
+	switch (mt->type) {
+	case PMEM_DEV_DAX: {
+		LOG(15, "pmem_persist addr %p, len %lu", addr, len);
+
+		pmem_persist(addr, len);
+
+		if (os_deep_flush_write(mt->region_id) < 0) {
+			LOG(2, "cannot write to deep_flush in region %d",
+				mt->region_id);
+			return -1;
+		}
+
+		return 0;
+	}
+	case PMEM_MAP_SYNC:
```

Comment:
either use braces everywhere or nowhere. This looks particularly jarring because you have 'return ...;' at different indents and at first glance it might look like you are missing a `break;`